### ISSUE TITLE
'BOT' n'est pas reconnu en tant que commande interne ou externe.

### ISSUE THREAD

**Issue by Vattory** [Androz2091/vinted-discord-bot#48]
Bonjour je suis sur windows 10 et j'ai cette erreur pouvez-vous m'aider ?

`Error: Command failed: BOT=Vattory VINTED_BOT_ADMIN_IDS=1048570303711821884 VINTED_BOT_TOKEN=MON TOKEN docker-compose -f docker-compose.yaml -p bot-Vattory up -d
'BOT' n'est pas reconnu en tant que commande interne
ou externe, un programme ex�cutable ou un fichier de commandes.

    at ChildProcess.exithandler (node:child_process:419:12)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1098:16)
    at ChildProcess._handle.onexit (node:internal/child_process:304:5) {
  code: 1,
  killed: false,
  signal: null,
  cmd: 'BOT=Vattory VINTED_BOT_ADMIN_IDS=1048570303711821884 VINTED_BOT_TOKEN=MON TOKEN docker-compose -f docker-compose.yaml -p bot-Vattory up -d'
}`

**Comment by BenjiBenji2** [comment]
contacte moi sur discord : Benji Benji#9688 
ça pourrais peut être t'intéressais pour résoudre ton problème

**Comment by LUZOOGG** [comment]
tu a trouver une solution?

**Comment by Vattory** [comment]
> tu a trouver une solution?

Oui, voici à quoi ressemble mon bot 
![image](https://github.com/Androz2091/vinted-discord-bot/assets/102914379/137871c9-fcb4-447a-94c9-c50ecafe72e7)

**Comment by LUZOOGG** [comment]
> > tu a trouver une solution?
> 
> Oui, voici à quoi ressemble mon bot ![image](https://user-images.githubusercontent.com/102914379/240036554-137871c9-fcb4-447a-94c9-c50ecafe72e7.png)

super peut tu me la donner?

**Comment by LUZOOGG** [comment]
> > tu a trouver une solution?
> 
> Oui, voici à quoi ressemble mon bot ![image](https://user-images.githubusercontent.com/102914379/240036554-137871c9-fcb4-447a-94c9-c50ecafe72e7.png)

super! peut tu me donner la solution?

**Comment by kennyyy53** [comment]
> > tu a trouver une solution?
> 
> Oui, voici à quoi ressemble mon bot ![image](https://user-images.githubusercontent.com/102914379/240036554-137871c9-fcb4-447a-94c9-c50ecafe72e7.png)

Est-ce que tu pourrais nous dire comment tu as résolu ton pb?

**Comment by Vattory** [comment]
> > > tu a trouver une solution?
> > 
> > 
> > Oui, voici à quoi ressemble mon bot ![image](https://user-images.githubusercontent.com/102914379/240036554-137871c9-fcb4-447a-94c9-c50ecafe72e7.png)
> 
> super! peut tu me donner la solution?

Oui, il faut modifier le module api-vinted qu'a créé Androz. Le site web de Vinted a beaucoup changé, et il est nécessaire de modifier les requêtes que l'API génère.

**Comment by Vattory** [comment]
> > > tu a trouver une solution?
> > 
> > 
> > Oui, voici à quoi ressemble mon bot ![image](https://user-images.githubusercontent.com/102914379/240036554-137871c9-fcb4-447a-94c9-c50ecafe72e7.png)
> 
> Est-ce que tu pourrais nous dire comment tu as résolu ton pb?

Oui, il faut modifier le module api-vinted qu'a créé Androz. Le site web de Vinted a beaucoup changé, et il est nécessaire de modifier les requêtes que l'API génère.

**Comment by DevRayro** [comment]
Mdr

> > > > tu a trouver une solution?
> > > 
> > > 
> > > Oui, voici à quoi ressemble mon bot ![image](https://user-images.githubusercontent.com/102914379/240036554-137871c9-fcb4-447a-94c9-c50ecafe72e7.png)
> > 
> > 
> > Est-ce que tu pourrais nous dire comment tu as résolu ton pb?
> 
> Oui, il faut modifier le module api-vinted qu'a créé Androz. Le site web de Vinted a beaucoup changé, et il est nécessaire de modifier les requêtes que l'API génère.

Super merci pour ton aide

**Comment by kazeyfr** [comment]
> > > > tu a trouver une solution?
> > > 
> > > 
> > > Oui, voici à quoi ressemble mon bot ![image](https://user-images.githubusercontent.com/102914379/240036554-137871c9-fcb4-447a-94c9-c50ecafe72e7.png)
> > 
> > 
> > Est-ce que tu pourrais nous dire comment tu as résolu ton pb?
> 
> Oui, il faut modifier le module api-vinted qu'a créé Androz. Le site web de Vinted a beaucoup changé, et il est nécessaire de modifier les requêtes que l'API génère.

pourrais-tu m'ajouter sur discord pour m'aider a resoudre le problème ?
